client/blocks/conversations/list.jsx

```diff
@@ -166,6 +167,47 @@ export class ConversationCommentList extends React.Component {
 		return { ...startingExpanded, ...expansions };
 	};
 
+	setActiveReplyComment = commentId => {
+		const siteId = get( this.props, 'post.site_ID' );
+		const postId = get( this.props, 'post.ID' );
+		this.props.setActiveReply( {
```

Comment:
should we bail if we couldn't find a `siteId` and `postId`? Or if that should never happen, should we drop the usage of `get`?

Reply:
👍 added in 7bfa279.